client/components/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@automattic/simple-components",
+  "version": "0.1.0-alpha.3",
+  "description": "React components, as used on WordPress.com",
+  "scripts": {
+    "build": "babel ./index.js --out-dir cjs && babel ./button --out-dir cjs/button && babel ./main --out-dir cjs/main",
+    "prepublishOnly": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/automattic/wp-calypso.git"
+  },
+  "keywords": [
+    "wordpress",
+    "react",
+    "components"
+  ],
+  "author": "Automattic",
+  "license": "GPL-2.0+",
+  "bugs": {
+    "url": "https://github.com/automattic/wp-calypso/issues"
+  },
+  "homepage": "https://github.com/Automattic/wp-calypso/blob/master/docs/components.md",
+  "main": "./cjs/index.js",
+  "esnext": "./index.js",
+  "files": [
+    "index.js",
+    "cjs/",
+    "button/",
+    "main/"
+  ],
+  "dependencies": {
+    "classnames": "1.1.1",
+    "lodash": "4.15.0",
+    "prop-types": "15.5.10",
+    "react": "15.4.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
```

Comment:
Yeah, because the `babel` CLI command is used in the [`build` script](https://github.com/Automattic/wp-calypso/pull/16782/files/78967e82d992eda630e05e1a14c35424a5057e7c#diff-37a3e92276c0c8d92e098bbcf201ec63R6), and without specifying a path, it's only guaranteed to be found when installed to `node_modules/.bin`